server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/modules/TestRabbitMQModule.java

```diff
@@ -69,6 +70,12 @@ protected RabbitMQConfiguration provideRabbitMQConfiguration() throws URISyntaxE
             .build();
     }
 
+    @Provides
+    @Singleton
+    private RabbitMQMailQueueConfiguration getMailQueueSizeConfiguration() {
+        return RabbitMQMailQueueConfiguration.sizeMetricsEnabled();
```

Comment:
No what happens if one guy don't get the file when starting James?

(That's not the classical way of handling missing configuration files)